### DISCUSSION TITLE
What are the chances of getting banned when using this?

### DISCUSSION THREAD

**Discussion by ySergi** [crunchy-labs/crunchy-cli#380] [Q&A] [answered]
I downloaded only one series of 12 episodes, but I'm afraid to download more...

**Comment by bytedream** [comment, marked as answer]
I personally didn't receive any sort of ban in the 2 years I'm developing the program. I also didn't hear from anybody else that they got banned

**Reply by ySergi** [reply]
Alright, that's a relief, thanks.
